src/index.html

```diff
@@ -11,7 +11,157 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset class="field">
+        <legend> Personal information </legend>
+
+        <label for="Surname" class="lab">Surname:
+          <input
+            type="text"
+            id="Surname"
+            name="user"
+            required
+          >
+        </label>
```

Comment:
you may not add `for` and `id` attributes when you wrap input with label
```suggestion
        <label class="lab">
          Surname:
          <input
            type="text"
            name="user"
            required
          >
        </label>
```